### DISCUSSION TITLE
Github Profil Page

### DISCUSSION THREAD

**Discussion by shadowcrypty** [shadowcrypty/shadowcrypty#4] [Q&A] [answered]
Hello everyone i start this discussion q&a for updating github profile pages.Help me to build much more attractive github profil page i want to hear  all you known  trick ,advice and suggestions.Also your can rewiev and comment about my github profile.

**Comment by furkannamli1** [comment, marked as answer]
👏

**Reply by shadowcrypty** [reply]
👏
